### ISSUE TITLE
Thermometer

### ISSUE THREAD

**Issue by zmwangx** [stanford-physics108-2015/cryo#7]
@marxwillia @rexgarland @bvlehm 

Thermometer seems to be our biggest roadblock at the moment (right?).

Today's discussion doesn't seem to work out, but I'm recording it here anyway. Here's link to Lake Shore's ruthenium oxide RTD product page: http://goo.gl/H1niqs . Below is a screenshot of the pricing (the calibration price for our desired range — or is it sensitivity? — is $3952):

![](http://i.imgur.com/2TVvXcy.png)

**Comment by zmwangx** [comment]
From yesterday's (01/16/2015) meeting with David and the TAs, it's likely that we'll purchase an uncalibrated model and either live with it or have Alex help us calibrate it (if calibration is necessary). I still have some doubt, but we'll see.

**Comment by zmwangx** [comment]
Forgot to mention: it was decided that ruthenium oxide is what we're gonna use.

**Comment by bvlehm** [comment]
Rick says that we have an uncalibrated Lakeshore RX-202A on order. Specs are [here](http://www.lakeshore.com/products/cryogenic-temperature-sensors/ruthenium-oxide-rox-rtds/models/pages/Specifications.aspx). TAs believe that calibration is not important above 0.1 K, so we will probably be able to use the thermometer uncalibrated.

**Comment by zmwangx** [comment]
Using the data provided by Lake Shore at http://goo.gl/RchRMO, I wrote a very simple module `r2t` to calculate temperature from resistance, available at https://github.com/stanford-physics108-2015/cryo/blob/master/py/r2t.py . I directly pushed it to master this time. In the future, should we go through pull requests (reviewed by at least one other person) before pushing to master?

I also added the tech specs and other data of the thermometer to the `instruments/` directory. We can add tech specs and stuff for other instruments when they become available.
